### PR TITLE
Move Isolate-related exports to sentry_io.dart

### LINKED ISSUE #1369: Version 7 broke web builds
```
$ webdev build
[WARNING] build_web_compilers:entrypoint on web/main.dart: Skipping compiling main|web/main.dart with dart2js because some of its
transitive libraries have sdk dependencies that are not supported on this platform:
sentry|lib/src/sentry_isolate_extension.dart
sentry|lib/src/sentry_isolate.dart
```
Note: Dart SDK `2.19.5`

### PR DESCRIPTION
**Pull request by mnordine**
Fixes https://github.com/getsentry/sentry-dart/issues/1369

### CODE REVIEW

**Comment by marandaneto** [comment]
@mnordine Can you address and provide more info as asked [here](https://github.com/getsentry/sentry-dart/issues/1369#issuecomment-1490257856)?

I'm not against the change but there's a trade-off here and I can't reproduce your issue.
People would need to import `sentry_io.dart` instead of `sentry.dart`.

**Comment by marandaneto** [comment]
@mnordine can you add a changelog entry?

**Comment by mnordine** [comment]
@marandaneto Do you want me to update `pubspec.yaml`, too?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 7.5.0
+
+### Fixes
+
+- Fix Dart web builds breaking due to `dart:io` imports when using `SentryIsolate` or `SentryIsolateExtension` ([#1369](https://github.com/getsentry/sentry-dart/pull/1371))
+- When using `SentryIsolate` or `SentryIsolateExtension`, import `sentry_io.dart`.
```
Comment (marandaneto) [inline CHANGELOG.md]:
```suggestion
## Unreleased

### Fixes

- Fix Dart web builds breaking due to `dart:io` imports when using `SentryIsolate` or `SentryIsolateExtension` ([#1369](https://github.com/getsentry/sentry-dart/pull/1371))
  - When using `SentryIsolate` or `SentryIsolateExtension`, import `sentry_io.dart`.
```

**Comment by marandaneto** [comment]
> @marandaneto Do you want me to update `pubspec.yaml`, too?

The release process does it automatically.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Unreleased
+
+### Fixes
+
+- Fix Dart web builds breaking due to `dart:io` imports when using `SentryIsolate` or `SentryIsolateExtension` ([#1369](https://github.com/getsentry/sentry-dart/pull/1371))
```
Comment (marandaneto) [inline CHANGELOG.md]:
```suggestion
- Fix Dart web builds breaking due to `dart:io` imports when using `SentryIsolate` or `SentryIsolateExtension` ([#1371](https://github.com/getsentry/sentry-dart/pull/1371))
```

**Review by marandaneto — APPROVED** [review]
Thanks @mnordine

**Pull request merged**